code/game/turfs/simulated/floor_types.dm

```diff
@@ -84,23 +92,22 @@
 	icon_state = "cult"
 
 
-/turf/simulated/floor/engine/n20
-	New()
-		. = ..()
-		var/datum/gas_mixture/adding = new
-		var/datum/gas/sleeping_agent/trace_gas = new
+/turf/simulated/floor/engine/n20/New()
+	..()
+	var/datum/gas_mixture/adding = new
+	var/datum/gas/sleeping_agent/trace_gas = new
 
-		trace_gas.moles = 2000
-		adding.trace_gases += trace_gas
-		adding.temperature = T20C
+	trace_gas.moles = 6000

```

Comment:
I think this was intentionally 2000, for a limited supply of n20, since it can be dangerous- *with ZAS.* But with LINDA, it's probably fine.